plugins/system/deborphan

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+: << =cut
+
+=head1 NAME
+
+deborphan - Monitor orphaned Debian packages
+
+=head1 APPLICABLE SYSTEMS
+
+Debian-ish systems with deborphan installed.
+
+=head1 CONFIGURATION
+
+None needed.
+
+=head1 AUTHOR
+
+Olivier Mehani <shtrom+munin@ssji.net>
+Based on the debsecan plugin by Nicolas Bouthors. 2016-09-02
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
+
+=cut
+
+# Auto enable if we have deborphan only
+if [ "$1" = "autoconf" ] ; then
+  if which deborphan >/dev/null; then
+    echo yes
+  else
+    echo "no (deborphan is missing)"
+  fi
+  exit 0
+fi
+
+# Fail if we don't have deborphan
+if ! which deborphan >/dev/null; then
+  echo "deborphan is missing" >&2
+  exit 1
+fi
+
+OUT=$(mktemp -t munin-deborphan.XXXXXX)
+TMPFILES=$OUT
+trap 'rm -f $TMPFILES' EXIT
```

Comment:
I would still prefer to not use files at all (avoiding any potential whitespace issue for `rm`), but this should work, if you really want to insist on it :)